builtin.py

```diff
@@ -58,6 +59,34 @@ def advance(self, index: int) -> Node:
     def __str__(self):
         return self.title
 
+class Obj(Node):
+    def __init__(self, title:str, desc:str, player:Player):
+        super().__init__(title, desc)
+        self.desc = desc
+        self.player = player
+        if title.isnumeric(): #initializing money
+            self.amount_Money = int(title)
+        elif title[:2] == "po": #if first 2 characters are po then its a potion
+            if title[2:] == "health":
+                self.amount_Hp = 25
+                self.potionType = "Health potion"
+
+class Potion(Obj):
+    def on_select(self):
+        if self in self.player.bag:
+            if self.amount_Hp:
+                self.player.hp += self.amount_Hp
+                self.player.bag.remove(self)
+        else:
+            self.player.bag.append(self)
+        for i in self.player.bag:
+            print(i.potionType)
+
+class Currency(Obj):
+    def on_select(self):
+        self.player.money += self.amount_Money
+        print(self.player.summary())
```

Comment:
I would consider making these classes inherit the Action class, which passes in the player already!

Reply:
We decided to keep the Items class because we agreed a divergence in the functionality eventually.